eng/pipelines/aggregate-reports.yml

```diff
@@ -122,15 +105,27 @@ stages:
               artifactName: reports
               path: '$(Build.ArtifactStagingDirectory)/reports'
 
-          - task: AzureFileCopy@2
-            displayName: 'Upload dependency report'
+          - task: AzureFileCopy@4
+            displayName: 'Upload dependency report (1)'
             inputs:
-              sourcePath: '$(Build.ArtifactStagingDirectory)/reports'
+              sourcePath: '$(Build.ArtifactStagingDirectory)/reports/*'
               azureSubscription: 'Azure SDK Artifacts'
               destination: AzureBlob
               storage: azuresdkartifacts
               containerName: 'azure-sdk-for-net'
               blobPrefix: dependencies
+              AdditionalArgumentsForBlobCopy: '--exclude-pattern=*data.js*'
+
+          - task: AzureFileCopy@4
+            displayName: 'Upload dependency report (2)'
```

Comment:
Why do we have 2 different uploads?

Reply:
its because am making copies to different directories int he blob container. I couldn't figure out how to use File Copy to do it in ione step.

Reply:
We should give the names more meaning beyond 1 and 2 and tell what they are for. For example it looks like one is reports and the other is graphs.